assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -398,6 +398,9 @@ che.infra.openshift.username=
 che.infra.openshift.password=
 che.infra.openshift.trust_certs=
 
+# Create routes with Transport Layer Security (TLS) enabled
+che.infra.openshift.enable_tls=false
```

Comment:
I'd rename it to `che.infra.openshift.tls_enabled`

Reply:
ok, will do